www/source/demo/windows/steps/10.html.slim

```diff
@@ -0,0 +1,24 @@
+section
+ h1 Next Steps
+
+ p Now that you have completed the web application guide, here are some additional resources to help you:
+
+ p <b>Documentation</b>
+
+ ul
+  li #{link_to 'Writing packages','/docs/developing-packages#sharing-pkgs'}
```

Comment:
The `#sharing-pkgs` anchor doesn't resolve for me. Should this just be `#{link_to 'Developing packages','/docs/developing-packages'}`?